nzedb/utility/Text.php

```diff
@@ -211,4 +211,90 @@ public static function convertRomanToInt($string)
 		}
 		return $e;
 	}
+
```

Comment:
Huh, I only added the extra line there though.  I moved it to Misc where we have another object2array function.